frontend/jest.config.js

```diff
@@ -3,6 +3,7 @@ module.exports = {
   globals: {
     'ts-jest': {
       tsconfig: '<rootDir>/tsconfig-test.json',
+      packageJson: 'package.json',
```

Comment:
ts-jest[config] (WARN) The option `packageJson` is deprecated and will be removed in ts-jest 27. This option is not used by internal `ts-jest`

Reply:
Oops, I can confirm this one. Remind me to actually try the stuff I review first ...

Reply:
Oh, I actually missed this one warning. I can try to reproduce the warnings, both on `master` and on this branch, and see if I missed somehting